test/kernel.js

```diff
@@ -1,24 +1,65 @@
-'use strict';
+"use strict";
```

Comment:
The style was not corrected in this file ;)
I forgot to add the test directory to the eslint command